ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/PostSSOTest.kt

```diff
@@ -134,5 +138,39 @@ class PostSSOTest : StringSpec() {
                 verifyBinding(finalHttpResponse)
             }
         }
+
+        "POST AuthnRequest With AuthnContext Test".config(enabled = runningDDFProfile()) {
```

Comment:
✏️ Add `DDF-Specific:` in the front. Here's an example from my PR: `"DDF-Specific: Redirect LogoutRequest With SHA256 Signature Test - Single SP"`